templates/static/js/src/demobox/components/LiveDemoBox/Select.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export type DropdownInfo = {
+  query: string;
+  label: string;
+};
+
+export type DropdownProps = {
+  dropdownOptions?: [DropdownInfo];
+  dropdownLabel?: string;
+  disabled?: boolean;
+};
+
+const Select = ({dropdownOptions, dropdownLabel, disabled}: DropdownProps) => {
+  return (
+    <>
+      <label htmlFor="exampleSelect" style={{opacity: disabled === true ? 0.5 : 1}}>
+        {dropdownLabel}
```

Comment:
Since `dropdownLabel` is an optional parameter, it might be `undefined` here. You can add a default value for it (like `""`), but it might be better to make it a mandatory parameter in the type definition above.

Reply:
Ahh yes, will do thanks!